### ISSUE TITLE
Filling zero after `optimizer.step()`

### ISSUE THREAD

**Issue by ray1007** [ray1007/pytorch-word2vec#4]
Once the bug in `pytorch` is fixed, I'll remove this line.

The bug is described in [pytorch forum](https://discuss.pytorch.org/t/bug-of-nn-embedding-when-sparse-true-and-padding-idx-is-set/9382/2) and reported in [github](
https://github.com/pytorch/pytorch/issues/3506).

**Comment by ssnl** [comment]
FYI, it is fixed today. But it is only fixed on master.

**Comment by ray1007** [comment]
Thanks! 
I'll fix this after a pip version is released. :)

Tzu-Ray

**Comment by luffycodes** [comment]
hello, i was wondering if the issue is fixed ?

thank you for the code :)
